### DISCUSSION TITLE
[Bug] languages not available in modal when one language in database

### DISCUSSION THREAD

**Discussion by henzeb** [area17/twill#859] [Q&A]
When you have translations enabled for a module, but have only one language record for a specific item, the language options are not visible in "editInModal" mode. 

This is the case for instance when you migrate existing tables that have no translations.

**Comment by ptrckvzn** [comment]
Hello @henzeb, can you provide a bit more details / steps to reproduce? What were you trying to do? Were you adding translations to an existing Model?
